### ISSUE TITLE
Add adfoc.us to debouncing rules

### ISSUE THREAD

**Issue by b-uwe** [brave/adblock-lists#1025]
Sample URL: http://adfoc.us/serve/sitelinks/?id=475250&url=http://optifine.net/adloadx?f=OptiFine_1.19.2_HD_U_H9.jar&x=4ec7

**Comment by fmarier** [comment]
Good idea once we have support for these kinds of rules in https://github.com/brave/brave-browser/issues/22429.
